app/views/shared/means_report/_item.html.erb

```diff
@@ -1,12 +1,25 @@
 <% addendum = local_assigns[:addendum] || nil %>
 <% suppress_border ||= false %>
 <% css_border_klass = suppress_border ? "govuk-summary-list__row--no-border" : "" %>
-<div class="<%= "govuk-summary-list__row  normal-word-break #{css_border_klass}" %>" id="<%= "means-merits-report__#{name}" %>">
+<div class="<%= "govuk-summary-list__row  normal-word-break" %>" id="<%= "means-merits-report__#{name}" %>">
```

Comment:
nit: `class="<%= "govuk-summary-list__row normal-word-break" %>"` are the erb opening and closing tags required or can they be removed?